staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -57,6 +57,9 @@ const (
 	// to the pod, which don't count towards the backoff limit, according to the
 	// pod failure policy. When the annotation is absent zero is implied.
 	JobIndexIgnoredFailureCountAnnotation = labelPrefix + "job-index-ignored-failure-count"
+	// JobControllerName reserved value for the managedBy field for the built-in
+	// Job controller.
+	JobControllerName = "job-controller.k8s.io"
```

Comment:
also needs the update to `kubernetes.io/job-controller` mentioned above

Reply:
Done